client/src/modules/Home/reducers/homeReducer.js

```diff
@@ -0,0 +1,10 @@
+const INITIAL_STATE = {};
+
+const homeReducer = (state = INITIAL_STATE, action) => {
```

Comment:
same here...looks like a reducer mock

Reply:
Yeah it was a reducer plug
I've already deleted all this unnecessary files